_data/docsarchive/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM starefossen/github-pages
+# The branch name to check out, also used in some find/replace
+# This is just the default, specify at build-time using --build-arg VER=<foo>
+ARG VER="v1.11"
```

Comment:
We need to "bake" the build-arg into the image so that it's available at runtime